File: Homeworks/C# Part 1/03.OperatorsAndExpressions/08.PrimeCheck/PrimeCheck.cs
using System;

class PrimeCheck
{
    static void Main()
    {
        int number = int.Parse(Console.ReadLine());
        bool isPrime = true;
        if (number < 2)
        {
            isPrime = false;
        }
        for (int i = 2; i < number; i++)
        {
            if (number % i == 0)
            {
                isPrime = false;
            }
        }
        Console.WriteLine(isPrime ? "true" : "false");
    }
}

File: Homeworks/JS Fundamentals/04. Operators and Expressions/07. Prime Check/prime-check.js
function solve(args) {
	var number = args[0];
	var isPrime = true;
	if (number < 2) {
		isPrime = false;
	}
	for (var i = 2; i < number; i += 1) {
		if (number % i === 0) {
			isPrime = false;
			break;
		}
	}	
	if (isPrime) {
		console.log('true');
	}
	else {
		console.log('false');
	}
}